data/releases/eol.yaml

```diff
@@ -1,4 +1,9 @@
 branches:
+  - release: "1.26"
+    finalPatchRelease: "1.26.16"
+    endOfLifeDate: 2024-02-28
+    note: >-
+      1.26.16 was released in March 2024 (after the EOL date) to pick up a new version of Go to [address several Go CVEs](https://groups.google.com/g/golang-dev/c/o1I1Vv8Rfgs/m/Wr8tD1RlAgAJ)
```

Comment:
Shouldn't it be `1.26.15`? See https://github.com/kubernetes/kubernetes/releases/tag/v1.26.15 (and there is 404 on https://github.com/kubernetes/kubernetes/releases/tag/v1.26.16)

Reply:
yes, good catch! There was a type-o in slack that I copied here. Updated.

Reply:
@jeremyrickard , Thank you for addressing this. Doesn't look like I can resolve the comment.